R/PlotEMODResults.R

```diff
@@ -89,32 +104,37 @@ emodplot.artcoverage <- function(data,
                          numerator = "On_ART",
                          denominator = "Infected")
   y.lim.max <- min(max(data$Prevalence) * 1.2, 1.0)
-  emodplot.by_gender(data, date.start, date.end, 'Prevalence', title=title ) +
-    scale_y_continuous(labels = scales::percent_format(accuracy = 1), breaks = seq(0, y.lim.max,0.25), limits=c(0,y.lim.max)) +
-    ylab("ART Coverage\n (% of Infected)")
 
+  p <- emodplot.by_gender(data, date.start, date.end, col2plot = 'Prevalence', title=title, plot.runs = plot.runs) +
+      scale_y_continuous(labels = scales::percent_format(accuracy = 1), 
+        breaks = seq(0, y.lim.max, max(ceiling(y.lim.max/5/.05)*.05, .05)), 
+        limits=c(0,y.lim.max)) +
+      ylab("ART Coverage\n (% of Infected)")
+    
+  return(p)
 }
 
 
 emodplot.incidence <- function(data,
-                            date.start,
-                            date.end) {
+                               date.start,
+                               date.end, 
+                               plot.runs = TRUE) {
   data.incidence <- EMODAnalyzeR::calculate.incidence(data)
   y.lim.max <- min(max(data.incidence$incidence) * 1.2, 1)
-  p <- emodplot.by_gender(data.incidence,date.start,date.end,'incidence') +
+  p <- emodplot.by_gender(data.incidence,date.start,date.end,'incidence', plot.runs = plot.runs) +
     scale_y_continuous(labels = scales::percent_format(accuracy = 1), breaks = seq(0,y.lim.max,0.005),limits=c(0,y.lim.max)) +
     ylab("HIV Incidence (%)")
   return(p)
 }
 
 
 emodplot.art <- function(data,
-                     date.start,
-                     date.end) {
-
-  data$On_Art_scaled <- data$pop_scaling_factor * data$On_ART
-  p <- emodplot.by_gender(data,date.start,date.end,'On_Art_scaled') +
-    ylab("Number on Art")
+                         date.start,
+                         date.end, 
+                         plot.runs = TRUE) {
+  data <- data %>% mutate(On_ART_scaled = pop.scaling.factor * On_ART)
```

Comment:
Code such as `calculate.pop_scaling_factor` expects the pop scaling factor to have the column name as `pop_scaling_factor` rather than `pop.scaling.factor`. Would you mind changing this back to `pop_scaling_factor`? FYI, this  caused an error in our test suite (in my opinion demonstrating the value of a test suite!).